stac_api/infrastructure/config.py

```diff
@@ -45,12 +46,13 @@ class vedaSTACSettings(BaseSettings):
     )
 
     userpool_id: Optional[str] = Field(
-        description="The Cognito Userpool used for authentication"
+        "None", description="The Cognito Userpool used for authentication"
```

Comment:
```suggestion
        None, description="The Cognito Userpool used for authentication"
```
Because this is cast as string in the construct does it need to be a string default?

Reply:
Nope, but it needs to have an explicit default, this is meant to just be `None`. Good catch!